css/at-rules/media.json

```
@@ -63,7 +63,8 @@
                 "version_added": "41"
               },
               "chrome_android": {
-                "version_added": "41"
+                "version_added": "41",
+                "notes": "Samsung devices incorrectly reports <code>(hover: hover)</code> on touch-screens. See <a href='https://crbug.com/940256'>bug 940256</a> and <a href='https://www.ctrl.blog/entry/css-media-hover-samsung.html'>more details</a>."
```

Just some quick grammar changes, as well as removing the blog post.  The Chrome bug is sufficient!
```suggestion
                "notes": "Samsung devices incorrectly report <code>(hover: hover)</code> on touchscreens. See <a href='https://crbug.com/940256'>bug 940256</a>."
```